storage_controller/src/http.rs

```diff
@@ -857,6 +857,21 @@ async fn handle_cancel_node_fill(req: Request<Body>) -> Result<Response<Body>, A
     json_response(StatusCode::ACCEPTED, ())
 }
 
+async fn handle_safekeeper_list(req: Request<Body>) -> Result<Response<Body>, ApiError> {
+    check_permissions(&req, Scope::Infra)?;
```

Comment:
`Scope::Admin` is always allowed. If we wanna call this from somewhere in the `infra` repo it should be `infra`.

Reply:
Ok. If it should be `Scope::Infra` then consider changing `handle_get_safekeeper` too -- seems kind of unnecessary to have separate grants for these.

Reply:
The node listing API also uses infra scope, which I used as inspiration for this handler. I think infra is better suited here, because there is the possible use case of calling this in an automated fashion from a place that is not the control plane, admin UI or developers using storcon_cli. But shrug.

Pushed a commit to also use the infra scope forthe get API.